packages/cli-kit/src/output.ts

```diff
@@ -141,6 +143,10 @@ export const token = {
   failIcon: () => {
     return new ContentToken('✖', {}, ContentTokenType.ErrorText)
   },
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  linesDiff: (value: any) => {
```

Comment:
I'm not very happy with using `any` here, but I've tried to make a generic type work without much success. I might take another stab at this later.

Reply:
Yeah the whole `class ContentToken` needs some attention and would probably be better served as a discriminated union